db/schema.rb

```diff
@@ -113,9 +114,9 @@
     t.integer "failed_attempts", default: 0, null: false
     t.string "unlock_token"
     t.datetime "locked_at", precision: nil
-    t.boolean "employed"
     t.datetime "remember_created_at", precision: nil
     t.string "remember_token"
+    t.boolean "employed"
```

Comment:
This and the change at 606 are not related to this PR.
It happens, some devs move them down and another commit moves them back it's weird and annoying, but not destructive